### PR TITLE
[feat] Add support for metrics query with last value

### PR DESCRIPTION
**Pull request by mihran113**

### CODE REVIEW

**Comment by gorarakelyan** [comment]
@mihran113 docs?

**Comment by mihran113** [comment]
> @mihran113 docs?

Added ✅

File: docs/source/using/search.md [hunk 1]
```diff
@@ -180,6 +180,7 @@ When iterating over metrics, use the `metric` keyword which represents the track
 | -------- | ----------- |
 | `name` | Metric name |
 | `context` | Metric context dictionary |
+| `values.[OPTION]` | Metric aggregated data about metric values such as `last`, `last_step` and etc. |
```
Comment (gorarakelyan) [inline docs/source/using/search.md:183]: should it be `metric.values.last_step` or `metric.steps.last`?

**Review by gorarakelyan — APPROVED** [review]

File: docs/source/using/search.md (hunk 1 above)
Comment (gorarakelyan) [reply docs/source/using/search.md:183]: and if the only supported option currently is `last`, then probably it is worth considering putting just `values.last`

**Review by alberttorosyan — APPROVED** [review]
looks good!

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Pull request merged**